src/globals/layout/_index.scss

```diff
@@ -4,8 +4,6 @@
 /// @copyright IBM Security 2019
 ////
 
-@import '../variables/index';
-
 @import '@carbon/layout/scss/layout';
```

Comment:
What's the point of this file

Reply:
😄

Reply:
This is a deprecation helper for consumers of older layout variables like `$spacing-*`, `get-fixed-size`, etc.